source/opt/ir_context.h

```diff
@@ -357,6 +357,13 @@ class IRContext {
   inline IteratorRange<std::multimap<uint32_t, Instruction*>::iterator>
   GetNames(uint32_t id);
 
+  // Returns an OpMemberName instruction that targets |struct_type_id| at
+  // index |index|. Returns nullptr if such instruction does not exist.
```

Comment:
replace "if such instruction does not exist" with "if no such instruction exists"